src/ConfigurationProvider.php

```diff
@@ -76,7 +76,7 @@ public function register(Application $app)
      */
     public static function isDev()
     {
-        return (new RuntimeSettings($_SERVER))->isDev();
+        return (new RuntimeSettings($_SERVER, []))->isDev();
```

Comment:
How come it doesn't use $_GET here? Also why doesn't this function use the DI version of RuntimeSettings? Does it have to be static?